frontend/src/styles/style.css

```diff
@@ -0,0 +1,9 @@
+#logo64 {
+  height: 64px;
+  width: 64px;
+}
+
+#logo256 {
+  height: 256px;
+  width: 256px;
```

Comment:
Is pixels the best way (units) to go with this?

Reply:
I decided to having the logo styled depending on use case instead of having fixed sizes like I originally did here.